utils/default-helpers/src/test/scala/pl/touk/nussknacker/engine/util/functions/GeoUtilsSpec.scala

```diff
@@ -0,0 +1,23 @@
+package pl.touk.nussknacker.engine.util.functions
+
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+
+class GeoUtilsSpec extends AnyFunSuite with Matchers {
+
+  test("accepts Null Island") {
+    geo.distanceInKm(0, 0, 0, 0) shouldBe 0
+  }
+
+  test("calculates distance between the White House and the Eiffel Tower") {
```

Comment:
Why not between the White House and the Emperor's hut :) It's a more contemporary example ;p

Reply:
You're onto something, but maybe let's hope that current name survives? ;)